libmscore/score.h

```diff
@@ -1283,7 +1283,7 @@ class MasterScore : public Score {
       bool isSavable() const;
       void setTempomap(TempoMap* tm);
 
-      bool saveFile();
+      bool saveFile(bool generateBackup = true);
```

Comment:
Files saved from within the plugin api should not genererate a backup, no?
So [this](https://github.com/Howard-C/MuseScore/blob/master/mscore/plugin/api/qmlpluginapi.cpp#L134) should add a `false` somewhere?
Maybe that should be on a musescore issue...

Reply:
If generateBackup is true the code isn't anything different from the way it is now. So anything else is a different issue.

Reply:
my error, it already doesn't generate any backup

Reply:
which I'd regard a bug

Reply:
@Jojo-Schmitz you think the plugin framework should save a backup of what it writes?
In its current implemenentation, it "saves a copy" of the score.

Reply:
If a plugin modifies an existing score, it should create a backup. If it does the equivalent of "Save as" it should not